E1-ALRF/ALIG/README.md

```diff
@@ -0,0 +1,177 @@
+# Test Instructions
+
+| Documentation Code   | Title                                          | Exchange Code | Test Code | Author          | Data Owner | Version | Date       |
+|----------------------|------------------------------------------------|---------------|-----------| ----------------|------------|---------|------------|
+| IFC4.3AbRV_E1-ALRF_ALIG   | Alignment Curves                     | E1-ALRF            | ALIG      | Sergej Muhic    | Let's see        | 1.0     | DD.MM.YYYY |
+
+
+## Summary
+
+*Include a short description of the test case.*
+
+*Optional: if your data comes from a specific Storyline or unit test of the IFC Infrastructure Deployment Project or IFC Rail Project, please include a link to the originating documentation here. so additional background information can be created.*
+
+With these instructions the baseline alignment exchange will be established between various domains. This includes basic alignment geometry featuring the common layout:
+- horizontal
+- vertical
+
+The valid geometry segment types for horizontal are the following:
+- Arc
+- Clothoid
+- Cubic (? has to be covered somewhere)
+- Linear
+
+The valid geometry segment types for vertical:
+- Arc
+- Linear
+- Parabolic (?does Road use it?)
+
+This test case addresses the import and export of *&&&&* entities within the *&&&* use case for the exchange of *&&&* information...
+
+...
+
+
+The [Expected Results](#Expected-Results) section lists the material that will be used to assess the fulfilment of capabilities.
+
+:zap: **This is a test-driven process: refer to the [Validation Criteria](#Validation-Criteria) to understand what is required by the test** :zap:
+
+## Itemised Roots
+The Test instruction addresses the import and export of the following IFC Entities & Concept Templates:
+
+<details><summary>IFC Entities</summary>
+
+These entities represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD. **The scope of the test shall not be used as a definitive scope of the exchange, or of the entire MVD.**
+
+    - *IfcAlignment*
+    - *IfcAlignmnetHorizontal*
+    - *IfcAlignmentVertical*
+    - *IfcAlignmentSegment*
+    - *IfcAlignmentHorizontalSegment*
+    - *IfcAlignmentVerticalSegment*
+    - *IfcAxis2PlacementLinear*
+    - *IfcAxis2Placement2D*
+    - *IfcAxis2Placement3D*
+    - *IfcCircle*
```

Comment:
How do we handle the cases in ALIN where we have no requirements for geometry at all (only layout)? Can we split this in two (with/without geometry)?